<ui version="4.0" >
 <class>WdgBracketing2HDR</class>
 <widget class="QWidget" name="WdgBracketing2HDR" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Source files:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidgetImages" >
     <column>
      <property name="text" >
       <string>Filename</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Exposure time</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>aperture</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Film sensitivity (iso)</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonAddImages" >
       <property name="text" >
        <string>add image</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Camera response:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGraphicsView" name="graphicsView" />
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>141</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCalculateCameraResponse" >
       <property name="text" >
        <string>Calculate camera response</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCreateHDRLayer" >
       <property name="text" >
        <string>Create HDR Layer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancel" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
